lib/blocks.js

```diff
@@ -949,6 +973,27 @@ var findNextNonspace = function() {
     this.indented = this.indent >= CODE_INDENT;
 };
 
+function measureNonspaceColumn(line) {
+    // This code is copied from findNextNonspace above
+    var i = 0;
+    var cols = 0;
+    var c;
+
+    while ((c = line.charAt(i)) !== "") {
+        if (c === " ") {
+            i++;
+            cols++;
+        } else if (c === "\t") {
+            i++;
+            cols += 4 - (cols % 4);
```

Comment:
Q: why must cols be a multiple of 4s?

Reply:
I think the answer is a tab is a multiple of 4?  not 2.

Reply:
Yeah, that's right. The Markdown spec treats a tab as if it's 4 spaces when that indentation may affect the structure of the document ([link](https://spec.commonmark.org/0.31.2/#tabs))